<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>216</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RoboClaw</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLabel" name="M2_act_pos_label">
      <property name="text">
       <string>Actual Position</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="M1_act_pos_label">
      <property name="text">
       <string>Actual Position</string>
      </property>
      <property name="buddy">
       <cstring>M1_act_pos_spinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="M1_act_pos_spinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="M1_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Motor 1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="M2_pos_spinBox">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="M2_pos_slider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="M2_act_pos_spinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="M2_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Motor 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="M1_pos_slider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="M1_pos_spinBox">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>M1_pos_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>M1_pos_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>M1_pos_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>M1_pos_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>M2_pos_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>M2_pos_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>M2_pos_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>M2_pos_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
